components/results/results.tsx

```diff
@@ -132,85 +133,107 @@ export const Results: FunctionComponent<ResultsProps> = (props) => {
-                                    title="play your moodqueue"
                                     text="queue"
                                     icon={<CirclePlay color="dark-2" />}
-                                    onClick={() => {
-                                        addToQueue(state.tracks)
-                                    }}
-                                />
-                                <Button
-                                    small
-                                    id="playlist-btn"
-                                    title="create a new moodqueue playlist"
-                                    text="playlist"
-                                    icon={<Playlist width="24px" height="24px" />}
-                                    onClick={() => {
-                                        addToPlaylist(state.tracks)
+                                    onClick={async () => {
+                                        await addToQueue(state.tracks)
+                                        resetForm()
                                     }}
-                                    secondary
                                 />
-                            </Box>
-                            <Button
-                                small
-                                id="reset-btn"
-                                title="start over to begin a new moodqueue"
-                                icon={<Previous color="light-2" />}
-                                text="back"
-                                onClick={resetForm}
-                                color="neutral-4"
-                            />
-                        </Box>
-                    ) : (
-                        <Box direction="row" align="center" gap="medium">
+                            ) : (
+                                <Box align="center">
+                                    <a data-for="queue-tooltip" data-tip data-event="click focus">
+                                        <Button
+                                            disabled
+                                            small
+                                            id="play-queue-btn"
+                                            text="queue"
+                                            icon={<CirclePlay color="dark-2" />}
+                                        />
+                                    </a>
+                                    <ReactTooltip
+                                        id="queue-tooltip"
+                                        globalEventOff="click"
+                                        effect="solid"
+                                    >
+                                        <Box width="small" align="center">
+                                            <Description
+                                                text="unfortunately, this feature is reserved for spotify premium users only :("
```

Comment:
i think we should say "limited to spotify premium users..." instead, we aren't like reserving it, we just are limited by the api